karma.conf.js

```diff
@@ -14,7 +14,7 @@ module.exports = function(config) {
       'libs/jasmine-jquery.js',
       'libs/angular.js',
       'libs/**/*.js',
-      'dist/**/*.js',
+      'src/**/*.js',
```

Comment:
Couldn't do `karma start` (watch) otherwise. We no longer test against the built version but the build only concats at the moment....